src/member/member.ts

```diff
@@ -22,14 +22,13 @@ export type MemberStorageItem = {
 };
 
 export type Pagination = {
-  totalItems: number;
-  totalPages: number;
-  currentPage: number;
+  totalCounts: number;
+  page: number;
   pageSize: number;
```

Comment:
I am still not convinced by the necessity to return the arguments `page` and `pageSize`. These were given in the request and they should not have been updated by the backend. @pyphilia @ReidyT @Defferrard please chime in to contradict me. 

If this is something we want to have, We should add it to the generic `Paginated` type and every paginated endpoint should return them.